### ISSUE TITLE
Display issue on #show when rendering from partial

### ISSUE THREAD

**Issue by superplussed** [sferik/rails_admin#2031]
Hello, I have an intermittent issue with displaying a partial for a field's value on the show pages.  If I include a partial like the following:

```javascript
field :job_assignment_histories do
  pretty_value do
    if bindings[:object].job_assignment_histories.present?
      bindings[:view].render(
        partial: "job_assignment_histories/table", 
        locals: {job_assignment_histories: bindings[:object].job_assignment_histories}
      )
    end
  end
end
```

I will occasionally get a list of the model's object reference like: 

JobAssignmentHistory#25, JobAssignmentHistory#26, JobAssignmentHistory#27

instead of getting the partial included.  

Is this related to pjax?  I can't set pjax to false on a show page, so how do I work around this other than turning off pjax altogether?

**Comment by glaukommatos** [comment]
I've run into a similar issue that I assume is related. My project is currently using rails_admin 0.6.5, and if an exception occurs in a pretty_value block I've supplied in my config/initializers/rails_admin.rb, then that pretty_value block will be ignored and rails_admin will instead render its default view for that field-type (in this case a has_many_association) for a time.

I do not believe it is strictly necessary that I reload the rails server before my own view will be rendered again for that particular field, but it does seem that restarting the server will fix it immediately. It seems that after a while, it will simply start rendering my custom pretty_value view again.

I'd love to know if anyone else has run into this issue. I've since fixed my pretty_value block to make it more resilient, but I'll admit I spent a very long time debugging this issue, since I wouldn't have expected my custom view to have been disabled for a period of time after an exception occurred, which is what seems like it happening.

I'd love to see some input from someone who's more familiar with how rails_admin chooses which views to load and who might be able to offer more insight into this behavior which I perceive as the view being temporarily disabled, because I'm not certain that I'm correctly interpreting what's happening. 

Thanks! This project is immensely useful and I'd love to help sort this out if it's actually a bug and not just me doing something wrong or misunderstanding something.

:+1:
